lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -189,13 +189,12 @@ export default function BasicLtiLaunchApp() {
 
   if (ltiLaunchState.state === 'fetched-url') {
     const iFrame = (
-      // FIXME-A11Y
-      // eslint-disable-next-line jsx-a11y/iframe-has-title
       <iframe
         width="100%"
         height="100%"
         className="js-via-iframe"
         src={ltiLaunchState.contentUrl}
+        title="Course assignment with annotation viewer"
```

Comment:
Can I suggest moving this change to a separate PR since I think it should be quicker to review and merge than the rest of the changes. I'm not certain about the best title here, maybe ask on Slack for suggestions.